### PR TITLE
[AIRFLOW-4129] Escape HTML in generated tooltips (#4944)

### PR DESCRIPTION
**Pull request by ashb**
Make sure you have checked _all_ steps below.

### Jira
https://issues.apache.org/jira/browse/AIRFLOW-4129

### Description

Escape anything that we expect to be plain text when building tooltips.

Many of these properties aren't user generated, but it is better to be
safe.

This is a back-port of #4944 to 1.10 branch on the RBAC and classic UI,
both of which are slightly different code layout to current master
Many of these properties aren't user generated, but it is better to be
safe

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

**Comment by ashb** [comment]
Please approve (if you are happy with this) but don't merge please.

**Review by mik-laj — CHANGES_REQUESTED** [review]
Hi.
You should not implement your own security mechanisms when it is not required. Look how you can achieve a similar effect, but using the browser's API.
https://codepen.io/anon/pen/moGGaE
Greetings

**Comment by kaxil** [comment]
Agree with @mik-laj

**Comment by ashb** [comment]
@mik-laj This is your version backported to 1.10. Could you give it a look over please?

File: airflow/www/templates/admin/master.html
Comment (mik-laj) [inline airflow/www/templates/admin/master.html]:
```suggestion
function escapeHtml(text) {
```
export is supported only in external modules.

Comment (ashb) [reply airflow/www/templates/admin/master.html]: Whops yes, thanks.

**Review by mik-laj — APPROVED** [review]

**Pull request merged**
